Project/Application/AppFlowCoordinator.swift

```diff
@@ -7,16 +7,55 @@ import UIKit
 
 final class AppFlowCoordinator: FlowCoordinator {
 
+    // MARK: - Constants
+    
+    private struct Constants {
+        static let homeString = "Home"
+        static let searchString = "Search"
+        static let profileString = "Profile"
+    }
+    
+    // MARK: - Properties
+    
     private let window: UIWindow
+    
+    private var homeFlowCoordinator: HomeFlowCoordinator
+    private var searchFlowCoordinator: SearchFlowCoordinator
+    private var profileFlowCoordinator: ProfileFlowCoordinator
 
+    // MARK: - Initializers
+    
     init(window: UIWindow) {
         self.window = window
+        searchFlowCoordinator = SearchFlowCoordinator()
+        homeFlowCoordinator = HomeFlowCoordinator()
+        profileFlowCoordinator = ProfileFlowCoordinator()
+        
+        super.init()
     }
 
     func initializeApp() {
-        let viewController = HelloWorldViewController(delegate: self)
-        rootViewController = UINavigationController(rootViewController: viewController)
-        window.rootViewController = rootViewController
+        let tabbarViewController = UITabBarController()
+        
+        let homeViewController = homeFlowCoordinator.rootViewController
+        let searchViewController = searchFlowCoordinator.rootViewController
+        let profileViewController = profileFlowCoordinator.rootViewController
+        
+        homeViewController.tabBarItem.title = Constants.homeString
+        searchViewController.tabBarItem.title = Constants.searchString
+        profileViewController.tabBarItem.title = Constants.profileString
+        
+        homeViewController.tabBarItem.image = UIImage(named: Constants.homeString)
```

Comment:
image of a view controller in tabbar should be responsibility of the view controller. FlowCoordinators manage routing, but not UI. Move these to init of view controller.

Reply:
Same with tabbar title